tests/hazmat/backends/test_openssl.py

```diff
@@ -434,6 +438,60 @@ def test_unsupported_oaep_label_decrypt(self):
                 )
             )
 
+    @pytest.mark.parametrize(
+        "vector",
+        load_vectors_from_file(
+            os.path.join("asymmetric", "RSA", "oaep-label.txt"),
+            load_nist_vectors)
+    )
+    @pytest.mark.skipif(
+        backend._lib.Cryptography_HAS_RSA_OAEP_LABEL != 1,
+        reason="Requires OpenSSL with OAEP label support (>= 1.0.2)")
+    def test_oaep_label_decrypt(self, vector):
+        private_key = serialization.load_der_private_key(
+            binascii.unhexlify(vector["key"]), None, backend
+        )
+        assert vector["oaepdigest"] == b"SHA512"
+        decrypted = private_key.decrypt(
+            binascii.unhexlify(vector["input"]),
+            padding.OAEP(
+                mgf=padding.MGF1(algorithm=hashes.SHA512()),
+                algorithm=hashes.SHA512(),
+                label=binascii.unhexlify(vector["oaeplabel"])
+            )
+        )
+        assert vector["output"].replace(b'"', b'') == decrypted
```

Comment:
We can drop the replace here if we modify the vector to remove the unneeded quotes.